Components/AddDestinations/AddDestinations.js

```diff
@@ -1,3 +1,4 @@
+import PropTypes from "prop-types";
```

Comment:
Great work, these small tests really help us.  I know we could test our app a lot more but right now every bit counts.